# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/posterazor/hotfix-cmake.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- src/CMakeLists.txt.vanilla	2026-01-28 15:38:38.113170019 +0100
+++ src/CMakeLists.txt	2026-01-28 15:38:48.879352903 +0100
@@ -1,3 +1,5 @@
+cmake_minimum_required(VERSION 3.5)
+
 PROJECT(PosteRazor)
 
 IF (WIN32 OR APPLE)
